### PR TITLE
Update Jitsi Meet to allow self-hosted versions and UserAgent fix

### PR DESCRIPTION
**Pull request by cybermoloch**
Updated Jitsi recipe to have hasCustomUrl: true. Left the default server in the serivceURL for a future version that may fill in that as a default server but would allow editing if 'hasCustomUrl' was true.

### CODE REVIEW

**Comment by cybermoloch** [comment]
Some notes here:

This should actually be Jitsi Meet as they have other "Jitsi" projects (Jitsi Desktop in the past so that comes to mind first when I hear Jitsi).

Leaving the serviceURL in there even with hasCustomUrl: true is necessary in this case of a switch over because otherwise Ferdi will just show "Loading Jitsi" forever with it switching. This is behavior should probably be documented in the recipe change/update docs if that is intended. (However, it does not show up in the configuration tab and will still insist it cannot be empty.)

**Comment by Nokia808** [comment]
Is there a hope to fix this & merge it ?

File: all.json [hunk 1]
```diff
@@ -444,7 +444,7 @@
     "featured": false,
     "id": "jitsi",
     "name": "Jitsi",
```
Comment (kytwb) [inline all.json:446]:
```suggestion
    "name": "Jitsi Meet",
```

File: uncompressed/jitsi/package.json [hunk 2]
```diff
@@ -1,13 +1,15 @@
 {
   "id": "jitsi",
   "name": "Jitsi",
```
Comment (kytwb) [inline uncompressed/jitsi/package.json:3]:
```suggestion
  "name": "Jitsi Meet",
```

File: uncompressed/jitsi/package.json [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
-  "version": "1.0.2",
+  "version": "1.0.3",
   "description": "Jitsi",
```
Comment (kytwb) [inline uncompressed/jitsi/package.json:5]:
```suggestion
  "description": "Jitsi Meet",
```

**Comment by kytwb** [comment]
@cybermoloch thank you for your contribution! 
I also updated the name of the recipe to "Jitsi Meet" as per your suggestion.

I merged despite the failing build as part of the refactor taking place in #292.

**Comment by kytwb** [comment]
Fixes https://github.com/getferdi/recipes/issues/149.

**Pull request merged**